01-smacss-media-queries/lab/labwork/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Lab 01 - SMACSS and Media Queries</title>
```

Comment:
In order for your site to be mobile responsive, make sure that you are including a viewport meta tag in the head:

`<meta name="viewport" content="device-width, initial-scale=1">`